cmd/doctor.go

```diff
@@ -26,11 +26,16 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type DoctorOptions struct {
```

Comment:
What is this struct for? A comment is good for all exported data structures and functions.

Reply:
On second glance it may not need to be exported?